gen/main.go

```diff
@@ -30,7 +30,7 @@ func main() {
 		bsky.EmbedExternal{}, bsky.EmbedExternal_External{},
 		bsky.EmbedImages_Image{}, bsky.GraphFollow{}, bsky.ActorProfile{},
 		bsky.EmbedRecord{}, bsky.FeedLike{}, bsky.RichtextFacet{},
-		bsky.RichtextFacet_ByteSlice{}, bsky.RichtextFacet_Features_Elem{},
```

Comment:
Interesting; we didn't need to serialize that type as part of post records?

Reply:
We do, but since they are unions they have to be codegenned by the lexgen command instead